File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/TunnelInfo.java
package com.test.pojo.entity;

import com.test.pojo.entity.TunnelInfo;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import javax.validation.constraints.NotNull;



@ApiModel("隧道信息")
@Getter
@Setter
public class TunnelInfo @extends {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("行别")
    private Integer direction;

    @ApiModelProperty("终点里程")
    private Float endMiles;

    @ApiModelProperty("隧道编号")
    private Integer id;

    @ApiModelProperty("输入码")
    private String inputCode;

    @ApiModelProperty("删除标记")
    @NotNull
    private Integer isDelete;

    @ApiModelProperty("全长")
    @NotNull
    private Float length;

    @ApiModelProperty("线路编号")
    @NotNull
    private Integer lineId;

    @ApiModelProperty("长链标记")
    @NotNull
    private Integer longChain;

    @ApiModelProperty("中心里程")
    @NotNull
    private Float middleMiles;

    @ApiModelProperty("操作人")
    @NotNull
    private Integer operator;

    @ApiModelProperty("备注")
    @NotNull
    private String remark;

    @ApiModelProperty("线别")
    @NotNull
    private Integer side;

    @ApiModelProperty("起点里程")
    @NotNull
    private Float startMiles;

    @ApiModelProperty("状态")
    @NotNull
    private Integer status;

    @ApiModelProperty("轨距")
    @NotNull
    private Float trackDistance;

    @ApiModelProperty("工务段编号")
    @NotNull
    private Integer trackDivisionId;

    @ApiModelProperty("隧道名")
    @NotNull
    private String tunneName;

    @ApiModelProperty("隧道号")
    @NotNull
    private String tunneNo;

    @ApiModelProperty("更新时间")
    @NotNull
    private Date updateTime;

}

File: src/tool/spring-boot-generator/output/railway/db/tunnelInfo.js
let tunnelInfo = {
	name: "tunnelInfo",
	description: "隧道信息",
	columns: {
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		direction: {
			name: "direction",
			description: "行别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		endMiles: {
			name: "endMiles",
			description: "终点里程",
			nullable: true,
			type: {
				name: "float"
			}
		},
		id: {
			name: "id",
			description: "隧道编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		inputCode: {
			name: "inputCode",
			description: "输入码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		length: {
			name: "length",
			description: "全长",
			nullable: true,
			type: {
				name: "float"
			}
		},
		lineId: {
			name: "lineId",
			description: "线路编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		longChain: {
			name: "longChain",
			description: "长链标记",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		middleMiles: {
			name: "middleMiles",
			description: "中心里程",
			nullable: true,
			type: {
				name: "float"
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		remark: {
			name: "remark",
			description: "备注",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		side: {
			name: "side",
			description: "线别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		startMiles: {
			name: "startMiles",
			description: "起点里程",
			nullable: true,
			type: {
				name: "float"
			}
		},
		status: {
			name: "status",
			description: "状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		trackDistance: {
			name: "trackDistance",
			description: "轨距",
			nullable: true,
			type: {
				name: "float"
			}
		},
		trackDivisionId: {
			name: "trackDivisionId",
			description: "工务段编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		tunneName: {
			name: "tunneName",
			description: "隧道名",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		tunneNo: {
			name: "tunneNo",
			description: "隧道号",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		}
	}
};

let columnsArray = [
	"createTime",
	"direction",
	"endMiles",
	"id",
	"inputCode",
	"isDelete",
	"length",
	"lineId",
	"longChain",
	"middleMiles",
	"operator",
	"remark",
	"side",
	"startMiles",
	"status",
	"trackDistance",
	"trackDivisionId",
	"tunneName",
	"tunneNo",
	"updateTime"
];

function create() {
  return {};
}

tunnelInfo.columnsArray=columnsArray;
tunnelInfo.create=create;
module.exports=tunnelInfo;